RedHatInsights/insights-inventory-frontend
# feat(ESSNTL-5214): Granular RBAC for actions & groups table

Implements https://issues.redhat.com/browse/ESSNTL-5214.

Granular rbac permissions for each rows actions and the bulk delete action.

how to test:
use account with granular groups write permissions + some groups without granular write permissions(only read permissions). Check the actions for the groups table
